### PR TITLE
fix(config): logic group config files

### LINKED ISSUE #2623: ZDI 5200 Param 5 duplicates param 4
https://github.com/zwave-js/node-zwave-js/blob/6f0cb2bbaf734c0bac2edd446853244de2bc89c7/packages/config/config/devices/0x0234/zdi5200.json#L39-L44

@Roagert you made this file - could you provide a fix for param 5? I found a manual, but I don't speak danish:
grafik
I _think_ this is about consumption reports.

### PR DESCRIPTION
**Pull request by Roagert**
zso7300 and zdi5200 fix
fixes zwave-js#2623

### CODE REVIEW

File: packages/config/config/devices/0x0234/templates/logic_group_template.json [hunk 1]
```diff
@@ -526,5 +526,70 @@
...
+			{
+				"label": "Always on",
+				"value": 1
+			},
+			{
+				"label": "Previous state",
+				"value": 2
+			}
+		]
+	},
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0234/templates/logic_group_template.json]: I don't think this is used anywhere.

File: packages/config/config/devices/0x0234/zso7300.json [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+			"label": "LED Indicator (Off): Green",
+			"valueSize": 4,
+			"defaultValue": 0
+		},
+		"7[0xff00]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "LED Indicator (Off): Blue",
+			"valueSize": 4,
+			"defaultValue": 255
+		},
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0234/zso7300.json]: The import `base_enable_disable` doesn't make sense here. It is a toggle for `Enable (1)` / `Disable (0)`.
I must have overlooked that last time.

**Review by AlCalzone — CHANGES_REQUESTED** [review]

**Comment by AlCalzone** [comment]
ping @Roagert I can't merge like this.

**Comment by Roagert** [comment]
> ping @Roagert I can't merge like this.

What you need me to do?

**Comment by AlCalzone** [comment]
See review comment: https://github.com/zwave-js/node-zwave-js/pull/2638/files/8d8a4384bf5633e5423338bbe24d9db464daee52#r634740232

**Review by AlCalzone — APPROVED** [review]

**Comment by AlCalzone** [comment]
Thanks!

**Comment by Roagert** [comment]
> Thanks!

You're welcome :)

**Pull request merged**